### ISSUE TITLE
#2 Error at install when reading the readme

### ISSUE THREAD

**Issue by novaws** [xmpppy/xmpppy#47]
OS: Windows 8.1 (RU)
Python: 3.8.2
xmpppy version: latest (0.6.2)

```
PS C:\Users\user\PycharmProjects\test\test> pip install xmpppy
Collecting xmpppy
  Using cached xmpppy-0.6.2.tar.gz (65 kB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\user\test\scripts\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-ughlxdiy\\xmpppy_7058d09e3db741378dd6e8812124d160\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-ughlxdiy\\x
mpppy_7058d09e3db741378dd6e8812124d160\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))'
 egg_info --egg-base 'C:\Users\user\AppData\Local\Temp\pip-pip-egg-info-hqjaxgvp'
         cwd: C:\Users\user\AppData\Local\Temp\pip-install-ughlxdiy\xmpppy_7058d09e3db741378dd6e8812124d160\
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-ughlxdiy\xmpppy_7058d09e3db741378dd6e8812124d160\setup.py", line 6, in <module>
        README = open(os.path.join(here, 'README.rst')).read()
      File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\lib\encodings\cp1251.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 1087: character maps to <undefined>
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/80/0f/df8de530e0cbe4956c3671a4db4365c85477096d5021d93125d01ea11854/xmpppy-0.6.2.tar.gz#sha256=5c1810d129f3f5ae5a2cf56791c9efcd393a34940c8efc75ad36cca40a699709 (from https://pypi.org/simple/xmpppy/). Command errored out with exit status 1: python setup.
py egg_info Check the logs for full command output.
  Using cached xmpppy-0.6.1.tar.gz (66 kB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\user\test\scripts\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-ughlxdiy\\xmpppy_0ad7323ff5ec4dffb6e7f5029f497ff3\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-ughlxdiy\\x
mpppy_0ad7323ff5ec4dffb6e7f5029f497ff3\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))'
 egg_info --egg-base 'C:\Users\user\AppData\Local\Temp\pip-pip-egg-info-umgfzmmn'
         cwd: C:\Users\user\AppData\Local\Temp\pip-install-ughlxdiy\xmpppy_0ad7323ff5ec4dffb6e7f5029f497ff3\
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-ughlxdiy\xmpppy_0ad7323ff5ec4dffb6e7f5029f497ff3\setup.py", line 6, in <module>
        README = open(os.path.join(here, 'README.rst')).read()
      File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\lib\encodings\cp1251.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 1093: character maps to <undefined>
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/25/34/42f1966f5c2ec33dc201d70ae71c255d510ef32288112e1b4c801aa9c011/xmpppy-0.6.1.tar.gz#sha256=48549adb7493facd85114c5cab9312e680417281a08038378306d371d791ea31 (from https://pypi.org/simple/xmpppy/). Command errored out with exit status 1: python setup.
py egg_info Check the logs for full command output.
  Using cached xmpppy-0.6.0.tar.gz (65 kB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\user\test\scripts\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-ughlxdiy\\xmpppy_9fb8b2f9adbb4511a43200675adfd105\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-ughlxdiy\\x
mpppy_9fb8b2f9adbb4511a43200675adfd105\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))'
 egg_info --egg-base 'C:\Users\user\AppData\Local\Temp\pip-pip-egg-info-990vw8g_'
         cwd: C:\Users\user\AppData\Local\Temp\pip-install-ughlxdiy\xmpppy_9fb8b2f9adbb4511a43200675adfd105\
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-ughlxdiy\xmpppy_9fb8b2f9adbb4511a43200675adfd105\setup.py", line 6, in <module>
        README = open(os.path.join(here, 'README.rst')).read()
      File "C:\Users\user\AppData\Local\Programs\Python\Python38-32\lib\encodings\cp1251.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 947: character maps to <undefined>
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/26/8a/2bf78d0246dae80497bfc9a22dcea9423e6e7c02ce5ea117612840e19704/xmpppy-0.6.0.tar.gz#sha256=4c01a306aecbc55d4c94f3157bad5ed3d14c1e69d7ee66754589cde7f69649c6 (from https://pypi.org/simple/xmpppy/). Command errored out with exit status 1: python setup.
py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement xmpppy (from versions: 0.5.0rc1, 0.6.0, 0.6.1, 0.6.2)
ERROR: No matching distribution found for xmpppy
PS C:\Users\user\PycharmProjects\test\test>
```

I confirm problem inside README.rst

**Comment by amotl** [comment]
Dear @novaws,

thank you for reporting this. I've just removed those special characters from the README file and released xmpppy version 0.6.3 to PyPI. Let me know if that works for you.

With kind regards,
Andreas.

**Comment by novaws** [comment]
> Dear @novaws,
> 
> thank you for reporting this. I've just removed those special characters from the README file and released xmpppy version 0.6.3 to PyPI. Let me know if that works for you.
> 
> With kind regards,
> Andreas.

yep. now all fine. very thx.

```
PS C:\Users\user\PycharmProjects\pythonProject> pip install xmpppy
Collecting xmpppy
  Downloading xmpppy-0.6.3.tar.gz (77 kB)
     |████████████████████████████████| 77 kB 1.3 MB/s
Building wheels for collected packages: xmpppy
  Building wheel for xmpppy (setup.py) ... done
  Created wheel for xmpppy: filename=xmpppy-0.6.3-py3-none-any.whl size=85960 sha256=7aa43963508f1db6dd27db1fc15497149707a83be1455d0d1033e296bccb653f
  Stored in directory: c:\users\user\appdata\local\pip\cache\wheels\96\a0\ef\2d38b3739de57a9b5740f60351853d2243cebb6301cf936b5b
Successfully built xmpppy
Installing collected packages: xmpppy
Successfully installed xmpppy-0.6.3
WARNING: You are using pip version 21.1.2; however, version 21.2.4 is available.
You should consider upgrading via the 'C:\Users\user\tunnel\pythonProject\Scripts\python.exe -m pip install --upgrade pip' command.
PS C:\Users\user\PycharmProjects\pythonProject>
```
